site/blog/2020-08-03-tool-fund-cambridge-neuro/README.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Analysis of spontaneous activity patterns in developing neural circuits using Frictionless Data tools
+date: 2020-08-03
+tags: ["tool-fund"]
+category: grantee-profiles
+image: /img/blog/fd_reproducible.png
+description: This grantee profile features Stephen Eglen for our series of Frictionless Data Tool Fund posts...
+author: Stephen Eglen
+---
+*This grantee profile features Stephen Eglen for our series of Frictionless Data Tool Fund posts, written to shine a light on Frictionless Data’s Tool Fund grantees, their work and to let our technical community know how they can get involved.*
+
+## Meet Stephen Eglen
+
+I am a Reader in Computational Neuroscience at the University of Cambridge.  A large part of my work involves analysing neuronal recordings taken from high-throughput recording devices, such as multielectrode arrays.  Despite these arrays having been in use for many years, there are still no standard formats for exchanging data, and so we spend lots of time simply reformatting data as we pass it around different groups, or use different analysis https://doi.org/10.1186/2047-217X-3-3 ) used HDF5; the aim of our current project is to evaluate the use of Frictionless Data as a common format for the analysis of our spontaneous activity recordings, both past and present.  The bulk of the work this summer will be done by a talented Natural Science undergraduate at Cambridge, Alexander Shtyrov. 
```

Comment:
Fixing minor typos, whitespace and convert link to Markdown format to avoid space before parenthesis:
```suggestion
I am a Reader in Computational Neuroscience at the University of Cambridge. A large part of my work involves analysing neuronal recordings taken from high-throughput recording devices, such as multi-electrode arrays. Despite these arrays having been in use for many years, there are still no standard formats for exchanging data, and so we spend lots of time simply reformatting data as we pass it around different groups, or use different analysis [https://doi.org/10.1186/2047-217X-3-3](https://doi.org/10.1186/2047-217X-3-3)) used HDF5; the aim of our current project is to evaluate the use of Frictionless Data as a common format for the analysis of our spontaneous activity recordings, both past and present. The bulk of the work this summer will be done by a talented Natural Science undergraduate at Cambridge, Alexander Shtyrov.
```